### PR TITLE
Add 1.18 versioned docs

### PR DESCRIPTION
**Pull request by codyjlandstrom**

### CODE REVIEW

File: src/content/release-notes.mdx [hunk 1]
```diff
@@ -5,6 +5,35 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.18.0
+
+27 January 2024
+
+This version is compatible with Kubernetes versions 1.25 to 1.28.
+
+### New Features
```
Comment (pchico83) [inline src/content/release-notes.mdx:14]: @codyjlandstrom I would call out the Argo CD installation guide. An alternative is to mentioning it in the release thena message, but I think it's important to let people know about this guide

**Review by pchico83 — APPROVED** [review]

File: src/content/release-notes.mdx (hunk 1 above)
Comment (codyjlandstrom) [reply src/content/release-notes.mdx:14]: Good call. Will add this now

File: src/content/release-notes.mdx [hunk 2]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release notes
...
+
+### Bug Fixes
+
+- Hide clear button if namespace search input is empty  <!-- 7667 -->
+- In deploy dialog, fix wrong filter when canceling current repository on SelectRepository  <!--7668 -->
+- Fix filtered namespace view where parent resources were not expanded  <!-- 7669 -->
+- Fix closing create access token dialog from keyboard  <!-- 7671 -->
+- Fix Scale to Zero Period showing wrong value in Previews UI  <!-- 7725 -->
+- Fix wrong "Waking Up" status when switching to a "Sleeping" namespace  <!-- 7817 -->
+- Properly render `code` nodes as inline elements inside a list item  <!-- 7821 -->
```
Comment (ifbyol) [inline src/content/release-notes.mdx:34]: Maybe we could add more detail indicating that this is for the customized readmes for external resources and possibly a link to https://www.okteto.com/docs/1.18/manifest/external-resources/#customizing-your-external-resource-ui-elements

**Review by ifbyol — COMMENTED** [review]
It seems the last commit failed to build but I wasn't able to finde a log error. Could it be related to the issues you were having locally?

File: src/content/release-notes.mdx (hunk 2 above)
Comment (codyjlandstrom) [reply src/content/release-notes.mdx:34]: Is that for all 4 of these items @ifbyol

Comment (ifbyol) [reply src/content/release-notes.mdx:34]: No, only for the one in the line 34 (`Properly render `code` nodes as inline elements inside a list item`)

**Pull request closed without merging**